Frameworks/CoreGraphics/CGPattern.mm

```diff
@@ -198,3 +209,24 @@ CGPatternRef CGPatternRetain(CGPatternRef pattern) {
     CFRetain((id)pattern);
     return pattern;
 }
+
+CGRect _CGPaternGetBounds(CGPatternRef pattern) {
```

Comment:
HOW!!!!

---
In reply to: [93077636](https://github.com/Microsoft/WinObjC/pull/1586#discussion_r93077636) [](ancestors = 93077636)